src/util/cert/libcrypto/cert.c

```diff
@@ -288,33 +373,84 @@ static errno_t ec_pub_key_to_ssh(TALLOC_CTX *mem_ctx, EVP_PKEY *cert_pub_key,
 #define SSH_RSA_HEADER "ssh-rsa"
 #define SSH_RSA_HEADER_LEN (sizeof(SSH_RSA_HEADER) - 1)
 
+static int sss_rsa_get_key(EVP_PKEY *cert_pub_key,
+                           BIGNUM **n, BIGNUM **e)
```

Comment:
According to the coding style output variables should start with an underscore.

Reply:
Fixed in the latest version.